Makefile

```diff
@@ -1161,6 +1161,7 @@ heat_cleanup: ## deletes the operator, but does not cleanup the service resource
 
 .PHONY: heat_deploy_prep
 heat_deploy_prep: export KIND=Heat
+heat_deploy_prep: export IMAGE=unused
```

Comment:
there's #217 that is supposed to address the problem for all the operators. 
Also, I think that patch set a global `IMAGE ?=unused` in the code, which 
is used by all the targets as a default, so we don't have to `export IMAGE=unused`
on each layer.
@kajinamit to comment further here.

Reply:
Yeah I think this should be fixed by #217 so we can use that instead of this one.

I didn't use heat_deploy_prep directly so didn't notice this. Good catch.

Reply:
Should we rebase this patch on top of #217 ?

Reply:
No. Once #217 is merged you don't have to set IMAGE=unused  because that is the internal default.